common/utils/src/main/resources/error/error-classes.json

```diff
@@ -451,6 +451,12 @@
     ],
     "sqlState" : "42704"
   },
+  "COLLATION_INVALID_NAME" : {
+    "message" : [
+      "The value <collationName> does not represent correct collation name."
```

Comment:
Could we propose some correct collation names that are similar to `collationName`? See for example:
https://github.com/apache/spark/blob/a3432428e760fc16610cfe3380d3bdea7654f75d/sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala#L109
or
https://github.com/apache/spark/blob/a3432428e760fc16610cfe3380d3bdea7654f75d/sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala#L319

Reply:
Done. Please take a look.